Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1168,7 +1380,67 @@ namespace xr
                         inputSource.AimSpace.Pose.Orientation.W = location.pose.orientation.w;
                     }
                 }
-                
+
+                // Get gamepad data 
+                const auto& controllerInfo = sessionImpl.ControllerInfo;
+                if (!controllerInfo.DefaultControllerBinding)
```

Comment:
nit (naming) - it makes more sense to me if you're checking to see if controllerInfo is using the MR controller binding, compared to checking if we aren't using the default bindings (since the code inside the conditional is specific to the MR bindings).